javascriptv3/example_code/personalize/src/personalize_createDomainDatasetGroup.js

```diff
@@ -0,0 +1,46 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createDomainatasetGroup.js demonstrates how to create a domain dataset group with Amazon Personalize.
+A domain dataset group is a dataset group containing preconfigured resources for different business domains and use cases.
+See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateDatasetGroup.html.
```

Comment:
For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateDatasetGroup.html.